docs/pages/workflow/android-studio-emulator.md

```diff
@@ -27,12 +27,18 @@ If you don't have an Android device available to test with, we recommend using t
 echo "export ANDROID_SDK=$ANDROID_SDK" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bash_profile'`
 ```
 
-- On macOS, you will also need to add `platform-tools` to your `~/.bash_profile` (or `~/.zshenv` if you use Zsh) - eg. `export PATH=/your/path/here:$PATH`. Copy and paste this line to do this automatically for Bash and Zsh:
+- On macOS, you will also need to add `platform-tools` to your `PATH` in `~/.bash_profile` (or `~/.zshenv` if you use Zsh) - eg. `export PATH=/your/path/here:$PATH`. Copy and paste this line to do this automatically for Bash and Zsh:
```

Comment:
maybe we should just make this read: "On macOS and Linux, you will also ..." and then when showing the line you copy and paste, provide an option for macos and another for linux. less repetition that way

Reply:
Sure, that makes sense